index.d.ts

```diff
@@ -0,0 +1,36 @@
+import { Model, ModelClass, Page } from 'objection';
```

Comment:
So this would have to be imported from `objection-2`. Plain ol' `objection` doesn't exist since we renamed our packages in `package.json`

Reply:
Noted